### DISCUSSION TITLE
I am unable to update my BlueRetro Cores.

### DISCUSSION THREAD

**Discussion by Mitch824** [darthcloud/BlueRetro#999] [Q&A]
I am unable to update my BlueRetro Cores. 
I keep getting an error message
app_ver_is_hw2: false app_name_is_hw2: false
I have tried to update them on 3 different computers and 2 different phones as well. I have also tried resetting the cores multiple times. 
Could anyone please help me?

**Comment by lamprey13** [comment]
Which "flavor" of the firmware are you using? I beleive you need to be using the HW1 version not the HW2 version.

**Comment by darthcloud** [comment]
Those things have a usb port,use that:
https://github.com/darthcloud/BlueRetro/wiki#82---via-usb-serial
